deployment/cdk/opensearch-service-migration/default-values.json

```diff
@@ -6,6 +6,7 @@
   "nodeToNodeEncryptionEnabled": true,
   "encryptionAtRestEnabled": true,
   "vpcEnabled": true,
+  "defaultFargateCpuArch": "X86_64",
```

Comment:
This doesn't seem like a setting that should be able to be modified as it depends on the image that's built (unless we change our build script to build within a container and be able to build for whichever target irregardless of host system).


I'd prefer this get auto-resolved, possibly through an `auto` setting here then using `const hostArchitecture = process.arch;`

See example for output on M1:

```bash
➜  TrafficCapture git:(main) ✗ node
Welcome to Node.js v18.18.2.
Type ".help" for more information.
> process.arch
'arm64'
```

and for X86 EC2:
```
[akurait@ip-10-0-3-66 ~]$ node
Welcome to Node.js v18.18.2.
Type ".help" for more information.
> process.arch
'x64'
```

Reply:
Good idea, have updated so as a default we will try to use `process.arch`